compute-aggregated.js

```diff
@@ -0,0 +1,38 @@
+require("dotenv").config();
+const fs = require("fs");
+const { Elm } = require("./compute-aggregated-app");
+
+const elmApp = Elm.ComputeAggregated.init({
+  flags: {
+    definitionsString: fs.readFileSync("public/data/impacts.json", "utf-8"),
+    textileProcessesString: fs.readFileSync("public/data/textile/processes_impacts.json", "utf-8"),
+    foodProcessesString: fs.readFileSync("public/data/food/processes_impacts.json", "utf-8"),
+  },
+});
+
+const exportJson = (filePath, json) => {
+  fs.writeFileSync(filePath, JSON.stringify(json, null, 2) + "\n");
```

Comment:
It would be nice to format the resulting JSON serialization with the prettier coding styles to avoid unwanted diffs; I did that for the examples editor here: https://github.com/MTES-MCT/ecobalyse/blob/3ca9f18f9a39b7e4bb8ed98d40eaff3a7e6c3fda/lib/github.js#L32-L34